refnx/reduce/platypusnexus.py

```diff
@@ -626,6 +627,118 @@ def _check_sample_environments(self, d, h5d):
         return d
 
 
+class SpinChannel(Enum):
+    """
+    Describes the spin state of a polarised neutron beam.
+    """
+
+    UPUP = (1, 1)
+    UPDOWN = (1, 0)
```

Comment:
is `UP_DOWN` preferable to `UPDOWN`? My preference would be to use the former throughout.